R/utils.R

```diff
@@ -87,10 +87,10 @@ plotRefMapToList <- function(map, plotRef) {
   # Validate naToZero and fix if necessary
   # NOTE failing to set naToZero will result in a default value of FALSE
   if (length(naToZero) == 0) {
-    warning("Encountered empty or NULL naToZero value. Setting naToZero = FALSE.")
+    #warning("Encountered empty or NULL naToZero value. Setting naToZero = FALSE.")
```

Comment:
are these intentionally commented out for moving forward or just helpful for testing?